ls1intum/Artemis
# K6: Exam Simulation

# Checklist

### Motivation and Context
This adds an exam simulation to the K6 test suite.

### Description
A new course is created with a new exam lasting 1 minute.
This exam contains 4 exercise groups, every exercise group has 1 exercise.
The 4 exercise types text, modeling, quiz and programming are used in the 4 exercise groups.
The specified number of users will be added to the exam and every 10th user (`userId % 10 == 0`) will get an extended working time of 3 minutes.
The students will take the exam and solve the exercises as long as possible.

Once the exam is over, the instructor will evaluate the quiz and delete the course.

The test is named `ExamAPIs`.

### Steps for Testing
1. Verify that the tests pass on Bamboo: https://bamboo.ase.in.tum.de/browse/ARTEMIS-SATJG2
